content/docs/command-reference/repro.md

```diff
@@ -10,14 +10,17 @@ analyzing dependencies and <abbr>outputs</abbr> of the target stages.
 ```usage
 usage: dvc repro [-h] [-q | -v] [-f] [-s] [-m] [--dry] [-i]
                  [-p] [-P] [-R] [--no-run-cache] [--force-downstream]
-                 [--no-commit] [--downstream] [--pull]
-                 [targets [targets ...]]
+                 [--no-commit] [--downstream] [--pull] [--glob]
+                 [targets [<target> ...]]
 
 positional arguments:
-  targets        Stage or path to dvc.yaml or .dvc file to reproduce. Using -R,
-                 directories to search for stages can also be given.
+  targets        Stage or path to dvc.yaml or .dvc file to reproduce.
+                 If no targets are provided, it is assumed to be the
+                 dvc.yaml present in the current working directory.
```

Comment:
Similar question @skshetry. I don't see this in master so I will change it to what's actually in the core repo unless you have more details. Thanks

Reply:
In fact I think I'm going to go for my earlier suggestion (https://github.com/iterative/dvc.org/pull/1983#discussion_r535430806):
```suggestion
  targets        Stage name or path to dvc.yaml or .dvc file to reproduce
                 (./dvc.yaml by default)
```
(and send a corresponding update to core)

Reply:
Hmmmm. Or "Limit command scope to these stage names." for consistency with all the other cmd refs that have a `targets` argument.

Reply:
> I don't see this in master so I will change it to what's actually in the core repo unless you have more details.

The previous `Using -R` wasn't in the DVC help output. I am not sure if they even should be, as the command reference and the help output has different objectives (check `git add -h` output). Like I said above, this is more suited for docs, rather than a help output.

Reply:
This part (the Synopsis) is a copy-paste of the help output (minus some details that would be repetitive). Sometimes it gets outdated, which is probably why the previous version also didn't match core.

> And again, Git output and docs aren't great. We do not try to copy the Git UI or git-scm.com at all.

> this is more suited for docs, rather than a help output

I disagree. At some point the UI and docs should intersect, and I think it's here. If it's clear enough for docs, it should be used in the UI as well. It also helps with consistency.